tests/test_bookmarks_updated.py

```diff
@@ -0,0 +1,127 @@
+"""
+Test tap sets a bookmark and respects it for the next sync of a stream
+"""
+from datetime import datetime as dt
+
+from dateutil.parser import parse
+
+from tap_tester import menagerie, connections, runner
+from base import BaseTapTest
+
+
+class BookmarkTest(BaseTapTest):
+    """Test tap sets a bookmark and respects it for the next sync of a stream"""
+    @staticmethod
+    def name():
+        return "tap_tester_shopify_bookmark_test"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.start_date = '2021-04-01T00:00:00Z'
+
+    def test_run(self):
+        with self.subTest(store="store_1"):
+            conn_id = self.create_connection(original_credentials=True)
+            self.bookmarks_test(conn_id, self.store_1_streams)
+
+        with self.subTest(store="store_2"):
+            conn_id = self.create_connection(original_properties=False, original_credentials=False)
+            self.bookmarks_test(conn_id, self.store_2_streams)
```

Comment:
I think we should start using a different pattern for parameterization than we have in the past. The unittest framework is intended to break up test cases by method. We can call two separate `test_` methods that both call the bookmarks test. This will improve test output by showing the actual number of tests running.

Reply:
done